src/test/java/homework_2/traffic_light/TrafficLightTest.java

```diff
@@ -0,0 +1,117 @@
+package homework_2.traffic_light;
+
+import base.UnitBase;
+import org.junit.jupiter.api.Test;
+
+
+import static homework_2.traffic_light.utils.MessageType.ERROR_MESSAGE_EXCEED_LIMIT;
+import static homework_2.traffic_light.utils.MessageType.ERROR_MESSAGE_INCORRECT_FORMAT;
+import static homework_2.traffic_light.utils.MessageType.ERROR_MESSAGE_NEGATIVE_NUMBERS;
+import static homework_2.traffic_light.utils.MessageType.INFO_MESSAGE;
+import static homework_2.traffic_light.utils.MessageType.INFO_MESSAGE_TO_MODE_0;
+import static homework_2.traffic_light.utils.MessageType.INFO_MESSAGE_TO_MODE_1;
+import static homework_2.traffic_light.utils.MessageType.LIGHT_GREEN_MESSAGE;
+import static homework_2.traffic_light.utils.MessageType.LIGHT_RED_MESSAGE;
+import static homework_2.traffic_light.utils.MessageType.LIGHT_YELLOW_MESSAGE;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TrafficLightTest extends UnitBase {
+
+    void run() {
+        new TrafficLight(0).start();
+        removeFromOutput(INFO_MESSAGE.getMessage());
+        removeFromOutput(INFO_MESSAGE_TO_MODE_0.getMessage());
+        printOut();
+    }
+
+    void runExtraMode() {
+        new TrafficLight(1).start();
+        removeFromOutput(INFO_MESSAGE.getMessage());
+        removeFromOutput(INFO_MESSAGE_TO_MODE_1.getMessage());
+        printOut();
+    }
+
+    @Test
+    void givenSecondsRemainderFrom0To34_whenRun_thenPrintGreenLight() {
+        setInput("315");
+        run();
+        assertEquals(LIGHT_GREEN_MESSAGE.getMessage(), getOutput());
```

Comment:
Opt: in general, it's bad practise to use constants from main. Tests should have their own, otherwise tests will not show you the change

Reply:
I added class Constants to the tests package.